cli/popper/commands/cmd_dot.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+import click
+import hcl
+import popper.utils as pu
+from ..cli import pass_context
+import types
+
+
+@click.argument(
+    'wfile', required=True)
```

Comment:
Can we make this mimic the behavior of `popper run`, where we look for `main.workflow` in the `$PWD`, or in `./github`. We might be able to move this functionality to `utils.py` so that `run` and `dot` can share it.

Reply:
Yeah, sounds good to me. Should I also move `-- recursive` functionality to `utils.py`, so `popper dot` can also use it?

Reply:
@ivotron let me know about moving the `--recursive` feature to `utils` so that both `run` and `dot` can use it

Reply:
@akashrajr1 yeah, that would be useful as well!

Reply:
Thanks, will update it soon

Reply:
Moved recursive feature to `find_recursive_wfile()` the in utils. It will now return a list of wfile locations, which can be iterated thrgh.